src/test/java/net/minecraftforge/test/CustomSpriteTest.java

```diff
@@ -0,0 +1,88 @@
+package net.minecraftforge.test;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.material.Material;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.renderer.texture.TextureAtlasSprite;
+import net.minecraft.client.resources.IResource;
+import net.minecraft.client.resources.IResourceManager;
+import net.minecraft.creativetab.CreativeTabs;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.client.event.TextureStitchEvent;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.event.FMLPreInitializationEvent;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
+import net.minecraftforge.fml.common.registry.GameRegistry;
+
+import javax.imageio.ImageIO;
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+
+/**
+ * Test for {@link TextureStitchEvent.Pre}.
+ */
+@Mod(modid = CustomSpriteTest.MOD_ID, name = CustomSpriteTest.NAME)
+public class CustomSpriteTest
+{
+    static final String MOD_ID = "custom_sprite_test";
+    static final String NAME = "Custom sprite test";
+
+    public void init(FMLPreInitializationEvent event)
```

Comment:
Please actually test, this doesn't even run. 
Missing `@Mod.EventHandler`
After I added that, it still didn't work properly.
![2016-11-27_17 10 03](https://cloud.githubusercontent.com/assets/916092/20653577/7475fb18-b4c4-11e6-86ab-681f285efddc.png)